python-nikola/PKGBUILD

```diff
@@ -1,27 +1,87 @@
+           'python-yapsy' 'python-pyrss2gen' 'python-dateutil' 'python-logbook'
+           'python-blinker' 'python-natsort')
+  depends+=('python-nikola-doc')
+  optdepends=('python-requests: for theme and plugin installation, and various web services'
+              'python-markdown: for Markdown support'
+              'python-jinja: for Jinja2 themes'
+              'python-livereload==2.3.2: for automatically refreshing browsers'
+              'python-pyphen>=0.9.1: for hyphenation'
+              'python-pygal==1.7.0: for SVG graph plotting'
+              'python-typogrify>=2.0.4: for typographical enhancements'
+              'python-webassets: for bundling assets'
+              'ipython-notebook: for ipynb support'
+              'ipython: for ipynb support'
+              'python-ghp-import>=0.4.1: for uploading to GitHub Pages'
+              'python-micawber: for embedding media in posts'
+              'python-phpserialize: for WordPress imports')
+  conflicts=('python-nikola-git')
+  cd "${srcdir}/${_pyname}-${pkgver}"
+  python3 setup.py install --prefix=/usr --root="${pkgdir}" --optimize=1
+  ln -s ${_pyname} "$pkgdir/usr/bin/${_pyname}3"
+  rm -rf "$pkgdir/usr/share"
+}
+
+package_python2-nikola() {
+  depends=('python2' 'python2-doit' 'python2-pygments' 'python2-pillow'
+           'python2-docutils' 'python2-mako' 'python2-unidecode' 'python2-lxml'
+           'python2-yapsy' 'python2-pyrss2gen' 'python2-dateutil' 'python2-logbook'
+           'python2-blinker' 'python2-natsort')
+  depends+=('python-nikola-doc')
+  optdepends=('python2-requests: for theme and plugin installation, and various web services'
+              'python2-markdown: for Markdown support'
+              'python2-jinja: for Jinja2 themes'
+              'python2-livereload==2.3.2: for automatically refreshing browsers'
+              'python2-pyphen>=0.9.1: for hyphenation'
+              'python2-pygal==1.7.0: for SVG graph plotting'
+              'python2-typogrify>=2.0.4: for typographical enhancements'
+              'python2-webassets: for bundling assets'
+              'ipython2-notebook: for ipynb support'
+              'ipython2: for ipynb support'
+              'python2-ghp-import>=0.4.1: for uploading to GitHub Pages'
+              'python2-micawber: for embedding media in posts'
+              'python2-phpserialize: for WordPress imports')
+  conflicts=('python2-nikola-git')
+  cd "${srcdir}/${_pyname}2-${pkgver}"
+  python2 setup.py install --prefix=/usr --root="${pkgdir}" --optimize=1
+  mv "$pkgdir/usr/bin/${_pyname}" "$pkgdir/usr/bin/${_pyname}2"
+  rm -rf "$pkgdir/usr/share"
+}
+
+package_python-nikola-doc(){
+  pkgdesc="nikola documentation license and man page"
+  conflicts=('python-nikola-doc-git')
+
+  cd "${srcdir}/${_pyname}-${pkgver}"
+  install -D -m644 LICENSE.txt "${pkgdir}/usr/share/licenses/${pkgbase}/LICENSE"
+  cd "docs/sphinx"
+  make SPHINXBUILD=sphinx-build html man
```

Comment:
don’t do this.  we do not support using this sphinx site in any way.  it’s up at https://docs.getnikola.com/ but we do not want users to build it themselves and use it as valid documentation